unabridged/motion-demos
# Add live preview

# Description

Add live preview.

The live preview demo was unstyled, and I couldn't resist styling it and adding an in-memory feed.  Let me know if you think that muddies the comparison.

# Testing
1. Go to '/live-preview' endpoint
2. Type in a tweet!  Submit it!  Add it!
a. On keypress, the live preview of the tweet should appear
b. On submit, the tweet should be added to the feed
c. On page reload, the 'feed' should disappear.  It is just in memory.
